floop-data-export.py

```diff
@@ -6,9 +6,14 @@
 
 # Get path of firebase cert from user. Prompt user if path doesnt exist
 while True:
-    certLoc = input('Enter full path of Floop Firebase cert file: ')
+    certLoc = 'floop-firebase-development-firebase-adminsdk-g8bpy-74eacd4956.json'
```

Comment:
Hard coding the certLoc variable inside the while loop runs into issues with the gPath statement that checks if 'certLoc' is a file. This results in the `if(gPath)' statement always returning false, and an infinite loop of line 20's print statement occurs.

Reply:
Hi, 
I have replaced the old code where the input location needs to be mentioned manually. I changed it because of our previous conversation where we were talking about hard-coding n
Will create pull requests once i confirm that my logic is correct